### PR TITLE
Dynamic workers edits

### PR DESCRIPTION
**Pull request by MJRichardson**
This was easier as a PR than as comments.

### CODE REVIEW

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 1]
```diff
@@ -4,35 +4,27 @@ description: Dynamic Worker pools are used in our cloud product to dynamically c
 position: 50
 ---
 
-Dynamic Worker Pools are a special type of [worker pool](/docs/infrastructure/workers/worker-pools.md) used by our Octopus Cloud to intialize a group of [workers](/docs/infrastructure/workers/index.md); when a task is assigned to a worker, the task will be executed by one of the workers in the pool.
+Dynamic Worker Pools are a special type of [worker pool](/docs/infrastructure/workers/worker-pools.md) used by Octopus Cloud.  They use [workers](/docs/infrastructure/workers/index.md) provided by Octopus, and don't require users to configure any infrastructure.  
 
-## Characteristics
+## On demand
 
-The Default Worker Pools are configured as a Dynamic Worker Pool, which means they have some different characteristics to older cloud instances and our self-hosted product.
```
Comment (MJRichardson) [inline docs/infrastructure/workers/dynamic-worker-pools.md:11]: I couldn't parse this sentence.

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 2]
```diff
@@ -41,25 +33,26 @@ If you're writing a script that relies on a specific version of tooling (e.g., h
...
 When an image is marked as deprecated, you will see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
 
 When you start getting warnings in your deployments and/or see deprecation warnings in the Octopus portal, please plan to modify your worker pool to use a newer image and test your scripts on the new image.
 
-### Batteries included
+## Available Dynamic Worker Images 
 
 Worker images are rebuilt on a regular basis, so they are up to date with the latest security patches.
 
-#### Windows Server Core 2016
```
Comment (MJRichardson) [inline docs/infrastructure/workers/dynamic-worker-pools.md:54]: I flipped 2016 and 2019 around

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 3]
```diff
@@ -92,9 +84,9 @@ Each `Windows Server Core 2019` worker is provisioned with a baseline of tools i
 - Powershell Core (latest)
 - Python (3.7.4)
 
-Windows 2019 workers are capable of running [execution worker containers](https://octopus.com/blog/execution-containers).
+Please note that [execution worker containers](https://octopus.com/blog/execution-containers) are not supported on Windows 2016 workers.
```
Comment (matt-richardson) [inline docs/infrastructure/workers/dynamic-worker-pools.md:87]:
```suggestion
Please note that [execution worker containers](/docs/deployment-process/execution-containers-for-workers/index.md) are not supported on Windows 2016 workers.
```

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 4]
```diff
@@ -70,18 +63,17 @@ Each `Windows Server Core 2016` worker is provisioned with a baseline of tools i
 - Powershell Core (latest)
 - Python (3.7.4)
 
-Please note that [execution worker containers](https://octopus.com/blog/execution-containers) are not supported on Windows 2016 workers.
+Windows 2019 workers are capable of running [execution worker containers](https://octopus.com/blog/execution-containers).
```
Comment (matt-richardson) [inline docs/infrastructure/workers/dynamic-worker-pools.md:66]:
```suggestion
Windows 2019 workers are capable of running [execution worker containers](/docs/deployment-process/execution-containers-for-workers/index.md).
```

File: docs/deployment-examples/kubernetes-deployments/kubectl/index.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+title: kubectl  
+description: The kubectl utility is required by Octopus Deploy kubernetes integration.  
+position: 100
+---
+
+## kubectl
+
+The [kubectl command-line tool](https://kubernetes.io/docs/reference/kubectl/overview/) is required by Octopus Deploy's kubernetes features.
+
+kubectl is not bundled with Octopus, and must be pre-installed on the [worker](/docs/infrastructure/workers/index.md) or Octopus Server which will execute steps and health checks against a [Kubernetes target](/docs/infrastructure/deployment-targets/kubernetes-target/index.md). Alternatively, [execution containers](/docs/deployment-process/execution-containers-for-workers/index.md) may be used.
```
Comment (matt-richardson) [inline docs/deployment-examples/kubernetes-deployments/kubectl/index.md:11]: i think execution containers can work, but they wont work for healthchecks i think

See this thread - https://octopusdeploy.slack.com/archives/C016SSDNTCY/p1597354774145800

**Review by matt-richardson — APPROVED** [review]
Nice - one comment 👍

File: docs/deployment-examples/kubernetes-deployments/kubectl/index.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: kubectl  
+description: The kubectl utility is required by Octopus Deploy kubernetes integration.
```
Comment (wordlee) [inline docs/deployment-examples/kubernetes-deployments/kubectl/index.md:3]:
```suggestion
description: The kubectl utility is required by Octopus Deploy Kubernetes integration.  
```

File: docs/deployment-examples/kubernetes-deployments/kubectl/index.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: kubectl  
+description: The kubectl utility is required by Octopus Deploy kubernetes integration.  
+position: 100
+---
+
+## kubectl
+
+The [kubectl command-line tool](https://kubernetes.io/docs/reference/kubectl/overview/) is required by Octopus Deploy's kubernetes features.
```
Comment (wordlee) [inline docs/deployment-examples/kubernetes-deployments/kubectl/index.md:9]:
```suggestion
The [kubectl command-line tool](https://kubernetes.io/docs/reference/kubectl/overview/) is required by Octopus Deploy's Kubernetes features.
```

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 8]
```diff
@@ -4,35 +4,27 @@ description: Dynamic Worker pools are used in our cloud product to dynamically c
 position: 50
 ---
 
-Dynamic Worker Pools are a special type of [worker pool](/docs/infrastructure/workers/worker-pools.md) used by our Octopus Cloud to intialize a group of [workers](/docs/infrastructure/workers/index.md); when a task is assigned to a worker, the task will be executed by one of the workers in the pool.
+Dynamic Worker Pools are a special type of [worker pool](/docs/infrastructure/workers/worker-pools.md) used by Octopus Cloud.  They use [workers](/docs/infrastructure/workers/index.md) provided by Octopus, and don't require users to configure any infrastructure.
```
Comment (wordlee) [inline docs/infrastructure/workers/dynamic-worker-pools.md:7]:
```suggestion
Dynamic Worker Pools are a special type of [worker pool](/docs/infrastructure/workers/worker-pools.md) used by Octopus Cloud.  They use [workers](/docs/infrastructure/workers/index.md) provided by Octopus, and they don't require users to configure any infrastructure.  
```

**Review by wordlee — APPROVED** [review]
I added a couple of minor edits. Looks good.

**Pull request merged**